pkg/process/process_vault.go

```diff
@@ -70,20 +70,22 @@ type VaultRoles struct {
 }
 
 type VaultRegionRoles struct {
-	DmeRoleID          string `json:"dmeroleid"`
-	DmeSecretID        string `json:"dmesecretid"`
-	RotatorRoleID      string `json:"rotatorroleid"`
-	RotatorSecretID    string `json:"rotatorsecretid"`
-	CtrlRoleID         string `json:"controllerroleid"`
-	CtrlSecretID       string `json:"controllersecretid"`
-	ClusterSvcRoleID   string `json:"clustersvcroleid"`
-	ClusterSvcSecretID string `json:"clustersvcsecretid"`
-	EdgeTurnRoleID     string `json:"edgeturnroleid"`
-	EdgeTurnSecretID   string `json:"edgeturnsecretid"`
-	AutoProvRoleID     string `json:"autoprovroleid"`
-	AutoProvSecretID   string `json:"autoprovsecretid"`
-	FrmRoleID          string `json:"frmroleid"`
-	FrmSecretID        string `json:"frmsecretid"`
+	DmeRoleID            string `json:"dmeroleid"`
+	DmeSecretID          string `json:"dmesecretid"`
+	RotatorRoleID        string `json:"rotatorroleid"`
+	RotatorSecretID      string `json:"rotatorsecretid"`
+	CtrlRoleID           string `json:"controllerroleid"`
+	CtrlSecretID         string `json:"controllersecretid"`
+	ClusterSvcRoleID     string `json:"clustersvcroleid"`
+	ClusterSvcSecretID   string `json:"clustersvcsecretid"`
+	EdgeTurnRoleID       string `json:"edgeturnroleid"`
+	EdgeTurnSecretID     string `json:"edgeturnsecretid"`
+	AutoProvRoleID       string `json:"autoprovroleid"`
+	AutoProvSecretID     string `json:"autoprovsecretid"`
+	FrmRoleID            string `json:"frmroleid"`
+	FrmSecretID          string `json:"frmsecretid"`
+	InferenceSvcRoleID   string `json:"inferencesvcroleid"`
+	InferenceSvcSecretID string `json:"inferencesvcsecretid"`
```

Comment:
I think the inference service can just use the cluster-svc role/secret IDs. They share the same effective role. Btw I assume the inference-svc will run in the regional cluster?

Reply:
correct, will run at the region level, since the apps are per-region.